webrtc-stats.html

```diff
@@ -1338,6 +1339,23 @@ <h2>
                   data for that frame.
                 </p>
               </dd>
+              <dt>
+                <dfn><code>totalInterFrameDelaySquared</code></dfn> of type <span class=
+                "idlMemberType">double</span>
+              </dt>
+              <dd>
+                <p>
+                  Sum of the squared interframe delay in seconds between consecutive decoded frames,
+                  recorded just after a frame has been decoded. The interframe variance can be
+                  approximately calculated from <a>totalInterFrameDelaySquared</a>,
+                  <a>framesDecoded</a>, and
+                  <a href="https://www.w3.org/TR/webrtc/#dom-rtcstats-timestamp">RTCStats.timestamp</a>
+                  by reading out these values at two timestamps <code>t0</code> and <code>t1</code>.
+                  This will be an approximation because <code>t0</code> and <code>t1</code> are
+                  timestamps of when the statistics are collected, which is only an approximation of
+                  when the last frame was decoded.
```

Comment:
I'm having trouble understanding if t0 and t1 need to be approximations of then the last frame was decoded, or if this is just a comment about the fact that we are (as always) working with averages. But I'll let other reviewers decide if this sentence is good or not. I'm happy with this PR now. Approved.

Reply:
One alternative would be to change _calculated_ to _estimated_ and remove the text about approximation.

Reply:
Not needed anymore, see the latest commit.